src/main/scala/com/resy/ResyApi.scala

```diff
@@ -108,7 +108,8 @@ object ResyApi extends Logging {
       .withHttpHeaders(
         createHeaders(resyKeys) ++ Seq(
           "Content-Type" -> "application/x-www-form-urlencoded",
-          "origin"       -> "https://widgets.resy.com/"
+          "Origin"       -> "https://widgets.resy.com",
+          "Referer"      -> "https://widgets.resy.com/"
```

Comment:
The extra `/` at the end of the Referer header is intentional to mirror how it works on the website.